### DISCUSSION TITLE
The — in the cheat sheet

### DISCUSSION THREAD

**Discussion by fredurb1** [KDE/ghostwriter#834] [Ideas]
Hello! I am using Ghostery to write novels, and it is what I needed. Thanks a lot!
But dialogs begin with the Em Dash — character, and it is not easy to get with a French keyboard. 
Would be great to insert easily "—" when needed, with a shortcut in the cheat sheet.

**Comment by wereturtle** [comment]
Hi @fredurb1 

To get an Em dash, type two hyphens, like so: `--`.

When the Markdown is converted to HTML or anything else, and smart typography is enabled, it will be converted into an Em dash.  :)

**Reply by fredurb1** [reply]
Cool! Thank you very much!

**Comment by aleksei-korolev** [comment]
If you are on Linux, you could enable compose keys. It's an OS-wide functionality and works in all apps. I prefer to bind it to the right Alt, for example. ComposeKey then "-" then "-" turns into "—", ComposeKey then "." then "." turns into "…". Research it for your OS. I had written this comment using compose keys.
